lib/reactive_table.html

```diff
@@ -83,7 +83,14 @@
         <div class="reactive-table-navigation">
           {{#if showNavigationRowsPerPage}}
             <div class="form-inline form-group rows-per-page">
-              <label>{{i18n 'reactiveTable.show'}}&nbsp;<input class="form-control" type="text" value="{{getRowsPerPage}}">&nbsp;{{i18n 'reactiveTable.rowsPerPage'}}</label>
+              <label>
+                <span class="show-label">{{i18n 'reactiveTable.show'}}</span>
+                <input class="form-control" type="text" value="{{getRowsPerPage}}">
+                {{#if showRowCount}}
+                  <span class="total-row-count">{{i18n 'reactiveTable.of'}} {{showRowCount}}</span>
+                {{/if}}
+                <span class="rowsPerPage-label">{{i18n 'reactiveTable.rowsPerPage'}}</span>
```

Comment:
Huh, I guess there are a few that don't follow it so these are okay if they're specific enough to the table. I do prefer dashes to camel case though. Don't change any that are already there in case people have customized them.